### PR TITLE
[WIP] Add "Banco ActivoBank" (PT) to list of banks with limited history

### PR DESCRIPTION
**Pull request by mtrocadomoreira**
---
category: Enhancements
authors: mtrocadomoreira
---

Add "Banco ActivoBank" (PT) to list of banks with limited history

### CODE REVIEW

**Comment by jfdoming** [comment]
Looks like there are merge conflicts. happy to approve after they are fixed!

**Comment by mtrocadomoreira** [comment]
Done!

File: src/app-gocardless/bank-factory.js [hunk 1]
```diff
@@ -99,6 +99,9 @@ export const BANKS_WITH_LIMITED_HISTORY = [
   'SEB_EEUHEE2X',
   'SEB_UNLALV2X',
   'SELLA_PERSONAL_CREDIT_SELBIT22',
+  'SANTANDER_BSCHESMM',
+  'NORDEA_NDEADKKK',
```
Comment (jfdoming) [inline src/app-gocardless/bank-factory.js:103]: Mind removing these two lines? They are duplicates of above

**Review by jfdoming — CHANGES_REQUESTED** [review]

File: src/app-gocardless/bank-factory.js (hunk 1 above)
Comment (mtrocadomoreira) [reply src/app-gocardless/bank-factory.js]: Sorry about that! I was careless with the merge. Done ✅

Comment (jfdoming) [reply src/app-gocardless/bank-factory.js]: All good!

**Review by jfdoming — APPROVED** [review]

**Pull request merged**